src/modules/todo-List.js

```diff
@@ -0,0 +1,115 @@
+import DynamicTodo from './dynamic-Todo';
+import CreateTodo from './create-Todo';
+import TodoStatus from './todo-Status';
+import { rotate, rotateReverse } from './rotate-RefreshBtn';
+
+const dynamicTodo = new DynamicTodo();
+
+class TodoList {
+  constructor() {
+    this.todoList = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  // Model
+  addTodo(todo) {
+    this.todoList.push(todo);
+    this.saveTodos();
+    rotate();
+    dynamicTodo.render(this.todoList, this);
+  }
+
+  removeTodo(todoId) {
+    this.todoList = this.todoList.filter(({ id }) => id !== todoId);
+    this.reindexTodos();
+    this.saveTodos();
+    rotateReverse();
+  }
+
+  // Sets Todo for Editing
+  setEditing(todoToEdit) {
+    this.todoList[todoToEdit - 1].isEditing = true;
```

Comment:
Consider implementing another function to ease the editing of saved tasks. The current option takes 2-steps with click operations.

Also being able to edit tasks on click will make it easier to edit tasks.